addons/mail/models/mail_scheduled_message.py

```diff
@@ -42,6 +42,8 @@ class MailScheduledMessage(models.Model):
     composition_comment_option = fields.Selection(
         [('reply_all', 'Reply-All'), ('forward', 'Forward')],
         string='Comment Options')  # mainly used for view in specific comment modes
+    notified_bcc = fields.Char('Bcc', compute='_compute_notified_bcc', readonly=True, store=False)
+    show_notified_bcc = fields.Boolean('Show BCC', default=False, store=False)
```

Comment:
False is always the default
```suggestion
    show_notified_bcc = fields.Boolean('Show BCC', store=False)
```